lib/server.js

```diff
@@ -364,10 +364,12 @@ function handleRequest(opts, req, resp) {
     .then(function(body) {
 
         if (body && /^application\/json/i.test(req.headers['content-type'])) {
-            try {
-                body = JSON.parse(body.toString());
-            } catch (e) {
-                reqOpts.log('error/request/json-parsing', e);
+            if (body.toString()) {
```

Comment:
Minor: Perhaps worth introducing a bodyString variable, to avoid the double conversion?